src/internal/dockertestenv/postgres.go

```diff
@@ -48,39 +48,36 @@ func NewTestDBConfig(t testing.TB) serviceenv.ConfigOption {
 		c.PostgresDBName = dbName
 
 		// direct
-		c.PostgresHost = postgresHost()
-		c.PostgresPort = postgresPort
+		c.PostgresHost = PostgresHost()
+		c.PostgresPort = PostgresPort
 		// pg_bouncer
-		c.PGBouncerHost = pgBouncerHost()
-		c.PGBouncerPort = pgBouncerPort
+		c.PGBouncerHost = PGBouncerHost()
+		c.PGBouncerPort = PGBouncerPort
 
 		c.PostgresUser = testutil.DefaultPostgresUser
 	}
 }
 
 func NewTestDB(t testing.TB) *pachsql.DB {
-	opts := NewTestDBOptions(t)
-	return testutil.NewTestDB(t, opts)
+	return testutil.OpenDB(t, NewTestDBOptions(t)...)
```

Comment:
Fixing a bug here because both `NewTestDBOptions()` and `NewTestDB()` call `CreateEphemeralDB()` which means this is creating two databases but we only use one.

Reply:
wow, the combination of name and signature in `NewTestDBOptions(t testing.TB) []dbutil.Option` seems almost deliberately misleading, not surprised that happened